x-pack/plugins/maps/public/classes/sources/mvt_single_layer_vector_source/mvt_single_layer_source_settings.test.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+jest.mock('../../../kibana_services', () => ({}));
+
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { MVTSingleLayerSourceSettings } from './mvt_single_layer_source_settings';
+import { MVT_FIELD_TYPE } from '../../../../common/constants';
+
+const defaultSettings = {
+  handleChange: () => {},
+  layerName: 'foobar',
+  fields: [
```

Comment:
Fields UI is already tested in its own snapshot. In this test, MVTFieldConfigEditor is just rendered as a shallow component. No reason to include more then a single entry or better yet, just an empty array.